Packs/Code42/Integrations/Code42/README.md

```diff
@@ -717,3 +717,53 @@ Reactivates the user with the given username.
 | UserID |
 | ------ |
 | 123456790 |
+### code42-legalhold-add-user
+***
+Adds a Code42 user to a legal hold matter.
+
+
+#### Base Command
+
+`code42-legalhold-add-user`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| username | The username of the user to add to the given legal hold matter. | Required | 
+| mattername | The name of the legal hold matter to which to which the user will be added. | Required | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Code42.LegalHold.UserID | Unknown | The ID of a Code42 user. | 
+| Code42.LegalHolder.MatterID | String | The ID of a Code42 legal hold matter. | 
+| Code42.LegalHold.Username | String | A username for a Code42 user. | 
+| Code42.LegalHold.MatterName | String | A name for a Code42 legal hold matter. | 
+
+
+#### Command Example
+```!code42-legalhold-add-user username="partner.demisto@example.com" mattername="test"```
+
+#### Context Example
+```
+{
+    "Code42": {
+        "LegalHold": {
+            "MatterID": "932880202064992021",
+            "MatterName": "test",
+            "UserID": "942876157732602741",
+            "Username": "partner.demisto@example.com"
+        }
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Code42 User Added to Legal Hold Matter
```

Comment:
I know that this happens from auto-generating, but it creates an extra cell and I think it's a bug... I don't see any other integration writing their human readable this way in the readme.